code/game/objects/structures/bedsheet_bin.dm

```diff
@@ -122,6 +126,8 @@ LINEN BINS
 /obj/item/weapon/bedsheet/brown/cargo
 	_color = "cargo"		//exists for washing machines, is not different from brown bedsheet in any way
 
+/obj/item/weapon/bedsheet/black
```

Comment:
why are you defining this twice? here and on line 96

Reply:
i'm a very tired man